src/libs/TransactionUtils/index.ts

```diff
@@ -1149,6 +1149,14 @@ function getTransactionViolationsOfTransaction(transactionID: string) {
     return deprecatedAllTransactionViolations?.[`${ONYXKEYS.COLLECTION.TRANSACTION_VIOLATIONS}${transactionID}`] ?? [];
 }
 
+/**
+ * Check if a transaction has been rejected
+ */
+function hasTransactionBeenRejected(transactionID: string): boolean {
```

Comment:
Does this check if the transaction has ever been rejected? Or just if it is currently rejected?